core/esmf-aspect-meta-model-java/src/main/java/org/eclipse/esmf/aspectmodel/edit/ChangeReport.java

```diff
@@ -16,22 +16,75 @@
 import java.util.List;
 import java.util.Map;
 
+import org.apache.jena.rdf.model.Model;
+
 /**
  * A structured representation of a number of {@link Change}s
  */
-public interface ChangeReport {
-   ChangeReport NO_CHANGES = new ChangeReport() {
-   };
+public sealed interface ChangeReport {
```

Comment:
If `permits` is left out for a sealed interface, all of its inner classes are automatically permitted. So I'd leave out the permits declaration when it's redundant (i.e., it would include exactly the set of inner classes).